memorix-recordtypes/Persoon.ttl

```diff
@@ -142,4 +150,27 @@ rt:Person
                            sh:maxCount              1 ;
                            sh:order                 2.0 ;
                            sh:path                  person:originallyFromLibrary ; ] ;
+    sh:property          [ rdfs:label               "Auteursrechtafspraken"@nl, "Copyright Agreement"@en  ;
+                           dash:editor              dash:TextAreaEditor ;
```

Comment:
Dit veld zou een linkedVocabulary moeten zijn: zoiets:

dash:editor             memorix:VocabularyEditor ;
memorix:conceptSchemeIn vocabularies:affe15be-8c77-40e5-87af-6e86137799e6 ;

verder perfect! Ik ga hetmorgen Rianne en co laten zien. Ben morgen om 10 uur in de Bazel, en  om 11 uur is het overleg. Misschien hebben we dan nig een uurtje om alles aan elkaar te knopen!